pysat/instruments/nasa_cdaweb_methods.py

```diff
@@ -260,27 +262,58 @@ def download(supported_tags, date_array, tag, sat_id,
         saved_local_fname = os.path.join(data_path, formatted_local_fname)
 
         # perform download
-        try:
-            print('Attempting to download file for '+date.strftime('%x'))
-            sys.stdout.flush()
-            remote_path = '/'.join((remote_url, formatted_remote_fname))
-            req = requests.get(remote_path)
-            if req.status_code != 404:
-                open(saved_local_fname, 'wb').write(req.content)
-                print('Finished.')
-            else:
+        if not multi_file_day:
+            try:
+                print('Attempting to download file for ' + date.strftime('%x'))
+                sys.stdout.flush()
+                remote_path = '/'.join((remote_url.strip('/'),
+                                        formatted_remote_fname))
+                req = requests.get(remote_path)
+                if req.status_code != 404:
+                    open(saved_local_fname, 'wb').write(req.content)
+                    print('Finished.')
+                else:
+                    print('File not available for ' + date.strftime('%x'))
+            except requests.exceptions.RequestException as exception:
```

Comment:
You should also output the exception message.

Reply:
Here and elsewhere

Reply:
fixed